tests/crypto/src/common_test.h

```diff
@@ -47,7 +42,23 @@ extern mbedtls_ctr_drbg_context ctr_drbg_ctx;
  *
  * @return								0 on success, else an error code.
  */
-int init_ctr_drbg(const unsigned char *p_optional_seed, size_t len);
+int init_drbg(const unsigned char *p_optional_seed, size_t len);
+
+#if defined(CONFIG_MBEDTLS_CTR_DRBG_C)
```

Comment:
Again, without CONFIG_

Reply:
8abda7e09d0c871e53353e762735ecb9426cd792

Reply:
could you squash that commit into https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/2164/commits/2bad95e00f85efd6ee188d6e895ce70920c7014d.
I see no reason for more than necessary commits in the PR

Reply:
I see https://github.com/NordicPlayground/fw-nrfconnect-nrf/commit/8abda7e09d0c871e53353e762735ecb9426cd792 covers more updates than just the touched lines. so never mind previous comment.